src/exchange-connector/resourceLoader/fatRamlResourceLoader.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import amf from "amf-client-js";
+import path from "path";
+import { FileSystemAdapter } from "./fileSystemAdapter";
+
+const EXCHANGE_MODULES = "exchange_modules/";
+
+export class FatRamlResourceLoader implements amf.resource.ResourceLoader {
+  public workingDir: string;
+  public fsAdapter: FileSystemAdapter;
+
+  constructor(rootFolder: string) {
+    this.workingDir = rootFolder.split("file://")[1];
```

Comment:
Can we assume that it will always be lowercase here? Or should we toLower it for the split?